Lab 5.ipynb

```diff
@@ -0,0 +1,595 @@
+       "                        <td id=\"T_4b9f7_row13_col2\" class=\"data row13 col2\" >-0.003412</td>\n",
+       "                        <td id=\"T_4b9f7_row13_col3\" class=\"data row13 col3\" >0.709354</td>\n",
+       "                        <td id=\"T_4b9f7_row13_col4\" class=\"data row13 col4\" >0.661513</td>\n",
+       "                        <td id=\"T_4b9f7_row13_col5\" class=\"data row13 col5\" >0.533815</td>\n",
+       "                        <td id=\"T_4b9f7_row13_col6\" class=\"data row13 col6\" >-0.466225</td>\n",
+       "                        <td id=\"T_4b9f7_row13_col7\" class=\"data row13 col7\" >0.263082</td>\n",
+       "                        <td id=\"T_4b9f7_row13_col8\" class=\"data row13 col8\" >-0.209706</td>\n",
+       "                        <td id=\"T_4b9f7_row13_col9\" class=\"data row13 col9\" >-0.675798</td>\n",
+       "                        <td id=\"T_4b9f7_row13_col10\" class=\"data row13 col10\" >-0.683419</td>\n",
+       "                        <td id=\"T_4b9f7_row13_col11\" class=\"data row13 col11\" >-0.621312</td>\n",
+       "                        <td id=\"T_4b9f7_row13_col12\" class=\"data row13 col12\" >-0.189381</td>\n",
+       "                        <td id=\"T_4b9f7_row13_col13\" class=\"data row13 col13\" >1.000000</td>\n",
+       "            </tr>\n",
+       "    </tbody></table>"
+      ],
+      "text/plain": [
+       "<pandas.io.formats.style.Styler at 0x7fa66da2faf0>"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "corr_coef = []\n",
+    "   \n",
+    "for key in qcd:\n",
+    "    for key2 in qcd:\n",
+    "        corr_coef.append([key, key2, stats.pearsonr(qcd[key], qcd[key2])])\n",
+    "\n",
+    "vals = []\n",
+    "for key in qcd:\n",
+    "    for key2 in qcd:\n",
+    "        for i in range(len(corr_coef)):\n",
+    "            if corr_coef[i][0] == key and corr_coef[i][1] == key2:\n",
+    "                vals.append(corr_coef[i][2][0])\n",
+    "\n",
+    "table_vals = []\n",
+    "for i in range(int(len(vals) / 14)):\n",
+    "    table_vals.append(vals[i*14:i*14 + 14])\n",
+    "\n",
+    "df = pd.DataFrame(table_vals, index = pd.Index(qcd.keys()), columns = pd.Index(qcd.keys()))\n",
+    "    \n",
+    "print(\"Correlation Coefficients-\")\n",
+    "df.style"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7374df60-2038-432b-ac73-6a839ff2d325",
+   "metadata": {},
+   "source": [
+    "Above is a table of the correlation coefficients for each measured value. This shows how much the data types align with each other with a linear relationship. The values are from -1 to 1, with 0 being no correlation, 1 being completely correlated (i.e. the same), and a negative value showing negative correlation as opposed to positive correlation. This is similar to what a person would qualitatively read from a 2D density histogram of the two data types."
```

Comment:
Eyy pandas! I like the format, might be good to explain what the correlation means before the table and/or include the formula.